source/game_sa/Audio/CAESmoothFadeThread.cpp

```diff
@@ -43,8 +43,10 @@ void CAESmoothFadeThread::Initialise()
         SetThreadPriority(m_threadHandle, 0);
         m_bThreadCreated = true;
         m_bThreadInvalid = false;
+        #if (WINVER >= 0x0A00)
```

Comment:
[Use better name?](https://docs.microsoft.com/en-us/cpp/porting/modifying-winver-and-win32-winnt?view=msvc-160)

```suggestion
        #if (WINVER >= _WIN32_WINNT_WIN10)
```

Reply:
i tested it with v141_xp toolset, win10 macro is not defined there

![image](https://user-images.githubusercontent.com/65975406/126702062-ea94d89f-6438-4766-ac15-3572be86c60b.png)

Reply:
Maybe something like that?
```C++
static constexpr bool CalcIsWin10() {
#ifdef _WIN32
    return WINVER >= 0x0A00;
#endif
    return false;
}

#define IsWin10 CalcIsWin10()
```